vignettes/lab_grading.Rmd

```diff
@@ -609,7 +627,7 @@ The decision made was put in the `COMMENT` field.
 <br>
 ```{r, eval=TRUE, echo=FALSE}
 atoxgr_criteria_ctcv5 %>%
-  filter(str_detect(TERM, "INR") | str_detect(TERM, "amylase")) %>%
+  filter(str_detect(TERM, "Lipase") | str_detect(TERM, "amylase")) %>%
```

Comment:
just checking - this was intentional here for INR?

Reply:
I was mistakenly filtering "INR" and "Serum amylase", I should have been filtering on "Lipase" and "Serum amylase".
INR was section before, so the danger of copy/pasting sections ;-)

Reply:
Thats what the update its, I have fixed it? Unless I misunderstand your original point?

Reply:
ah okay.  just making sure